types/react-places-autocomplete/index.d.ts

```diff
@@ -38,6 +39,7 @@ export interface PropTypes {
         location?: google.maps.LatLng | google.maps.LatLngLiteral;
         offset?: number | string;
         radius?: number | string;
+        sessionToken?: any;
```

Comment:
Better to type using:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a8fb783542bd3e38682182ef8fd9225215b2dc19/types/googlemaps/index.d.ts#L3608

Reply:
According to the documentation, it is not strictly required to use AutocompleteSessionToken. It is just there as a way to make life easier, but anything unique (like a self generated UUID) will suffice as token.